### PR TITLE
Added a Bash-client for Pantry API

### PR DESCRIPTION
**Pull request by SomajitDey**
Just discovered your wonderful work. I needed a free and simple json store for my projects and getpantry.cloud turned out to be perfect. Made a client for myself. Thought others might find it useful. Hope you like it.

### CODE REVIEW

File: client-libraries/Bash/pantry-client.bash [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Library of bash-functions to access https://getpantry.cloud/ API
+# Just remember to put your Pantry id below.
+# Author: Somajit Dey <dey.somajit@gmail.com> 2021
+
+export PANTRY_ID="cc9c05f7-ce68-4e90-9dfa-235f2515504e"
```
Comment (imRohan) [inline client-libraries/Bash/pantry-client.bash:5]: Replace your personal PantryID with "Your_Pantry_ID_here"

**Review by imRohan — CHANGES_REQUESTED** [review]
Thanks for putting in the great work! Just one minor comment, but other than that it looks great!

File: client-libraries/Bash/pantry-client.bash (hunk 1 above)
Comment (SomajitDey) [reply client-libraries/Bash/pantry-client.bash]: Oops! Took a mental note to do that after completing the test and then forgot ;-)

In the latest commit I have changed things a little. Now PANTRY_ID needs to be passed as parameter while sourcing the script. If the user forgets, there would be warnings. Hope it's more user-friendly now.

Cheers.

File: client-libraries/Bash/pantry-client.bash [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+#
+# Tip: Put the above command inside ~/.bashrc to avoid sourcing manually everytime
+#
+# Author: Somajit Dey <dey.somajit@gmail.com> 2021
+
+export PANTRY_ID="${1:-"${PANTRY_ID}"}"
+
+[[ -z "${PANTRY_ID}" ]] && \
+echo 'Warning: PANTRY_ID not set
+Execute command: export PANTRY_ID="Your_Pantry_ID_here"'
```
Comment (imRohan) [inline client-libraries/Bash/pantry-client.bash:15]: Nice!

**Review by imRohan — APPROVED** [review]
Looks good! Please squash your commits. I will merge this PR in once that's been completed.

File: client-libraries/Bash/pantry-client.bash (hunk 2 above)
Comment (SomajitDey) [reply client-libraries/Bash/pantry-client.bash:15]: Thank you so much. I have squashed all my commits too.

**Pull request merged**